.github/workflows/agw.main.yml

```diff
@@ -0,0 +1,43 @@
+name: main-branch
+
+on:
+  pull_request:
```

Comment:
Can we also run it on pushes?

Reply:
Yes, in fact I thought about the same in the evening.